worker/src/constants.rs

```diff
@@ -40,3 +40,9 @@ pub static EXTRINSIC_MAX_SIZE: usize = 4196;
 pub static STATE_VALUE_MAX_SIZE: usize = 1024;
 // Iterator for RocksDB returns these many elements in a single yield
```

Comment:
I guess it's just a comment, but as we are not using RocksDB anymore this could be changed.

Reply:
Fixed